src/components/views/dashboard.tsx

```diff
@@ -76,6 +77,31 @@ export const Dashboard: FC<any> = ({navigation}) => {
     />
   );
 
+  const type = exposure.status.type ? exposure.status.type : [];
+
+  const renderAlertInfo = () => {
```

Comment:
Are you sure this covers all cases?

Reply:
@floridemai This does seem to fix the main issue on Android, the one case I can find that isn't covered is:

 - Have the app open on any tab with ENS disabled for any reason
 - Put the app in the background.
 - Cause ENS to become enabled (e.g. re-enable disabled bluetooth)
 - Focus the app. If on another tab, tab over to dashboard. On reaching the dashboard, the yellow card is there for a second or two until the `useFocusEffect` completes.

This seems to be because the `exposure.status.state` remains `'disabled'` even though `exposure.enabled` is true, until the async calls in the useFocusEffect fire. 

I don't see any clean way around that however, without some major changes to this logic which has been very thoroughly tested.

I think it's only a UX problem if the user was on a screen other than `dashboard`, then navigates over to the dashboard. If they were already in the dashboard, the yellow card is already in the preview and the last screen they saw so it's not so much a flash as a confirmation that they did indeed fix the ENS. Maybe we could call `exposure.readPermissions` on a listener to the app coming into focus so that in this one case the card removes itself before the user navigates over to the dashboard.

Reply:
I have my doubts, but will need to test it. I'll find some time and come back with my findings.